lib/PuppeteerSharp.Tests/PuppeteerSharp.Tests.csproj

```diff
@@ -18,6 +19,10 @@
   <PropertyGroup Condition="'$(TargetFramework)'=='net471'">
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">

```

Comment:
what are these for?

Reply:
Damned. I have to revert this. My VS is hanging when I try to debug test. I'm trying to see what's going on. I accidentally committed this.

Reply:
Reverted.